man/systemd-stub.xml

```diff
@@ -142,11 +144,25 @@
       details on system extension images. The generated <command>cpio</command> archive containing these
       system extension images is measured into TPM PCR 13 (if a TPM is present).</para></listitem>
 
+      <listitem><para>Similarly, files
+      <filename><replaceable>foo</replaceable>.efi.extra.d/*.addon.efi</filename>
+      are loaded and verified as PE binaries, and a <literal>.cmdline</literal> section is parsed from them.
+      In case Secure Boot is enabled, these files will be validated using keys in DB or MoK, and will be
+      rejected otherwise. This is supposed to be used to pass additional kernel command line parameters,
+      regardless of the kernel image being booted, for example to allow platform vendors to ship
+      platform-specific configuration. The loaded command line options are sorted and measured into TPM PCR
```

Comment:
it's not the options that are sorted, but the files the options are loaded from. and the option strings concatenated.

should also say what happens to the cmdline embedded in the UKI itself if an addon is discovered.